src/lib/netlist/plib/ptypes.h

```diff
@@ -90,7 +90,8 @@ namespace plib
 	{
 		UNKNOWN,
 		LIBSTDCXX,
-		LIBCPP
+		LIBCPP,
+		MSC
```

Comment:
The name of the Microsoft C++ runtime is `MSVCPRT` (from **M**icrosoft **V**isual **C** **P**lus plus **R**un**T**ime.